app/src/main/java/org/oppia/app/player/content/ContentViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.app.player.content
+
+import androidx.databinding.ObservableField
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.model.SubtitledHtml
+import javax.inject.Inject
+
+/** [ViewModel] for content-card state. */
+@FragmentScope
+class ContentViewModel @Inject constructor(
+) : ViewModel() {
+
+  val contentId = ObservableField<String>("content_id")
+  val html_content = ObservableField<String>("html")
```

Comment:
We are following camelcase convention.

`htmlContent`

Reply:
changed to htmlContent